cli/tests/test_split_command.rs

```diff
@@ -346,8 +346,8 @@ fn test_split_siblings_no_descendants() {
     let (stdout, stderr) = test_env.jj_cmd_ok(&workspace_path, &["split", "--siblings", "file1"]);
     insta::assert_snapshot!(stdout, @"");
     insta::assert_snapshot!(stderr, @r###"
-    First part: qpvuntsm?? 8d2b7558 TESTED=TODO
```

Comment:
I'm not sure why this was marked with a `??` before or why it's gone now. I guess the change became divergent somehow due to a bug in the old rebasing logic?

Reply:
Looks like it was only divergent at the time when we printed the summary and it got fixed later during `tx.finish()` which calls `rebase_descendants()`.

The bug is that if the commit has no children then we don't call `rebase_descendants()` when `--siblings` is used.

https://github.com/martinvonz/jj/blob/77eaf67f96c6ea438e2dd1a5636f34b6a321e6be/cli/src/commands/split.rs#L223-L235

That's fixed after this PR.